### PR TITLE
Add pill component

### PR DESCRIPTION
**Pull request by dominik-zeglen**
This PR adds new pill component.

### Screenshots
Zrzut ekranu 2022-01-18 o 15 00 38
Zrzut ekranu 2022-01-18 o 15 00 50

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4833 Rework pills](https://app.clickup.com/t/2549495/SALEOR-4833)

File: src/Pill/Pill.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const guideClasses = useGuideStyles();
+
+  return (
+    <div>
+      <Typography className={guideClasses.headline} variant="h1">
+        Pills
+      </Typography>
+      <Typography className={guideClasses.paragraph} component="p">
+        Pills are used to display certain status of a given entity. It could be
+        paid or unpaid order, a VAT number awaiting verification, or plugin that
```
Comment (Cloud11PL) [inline src/Pill/Pill.stories.tsx:35]:
```suggestion
        paid or unpaid order, a VAT number awaiting verification, or a plugin that
```

File: src/theme/ThemeProvider.tsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({
   palettes = {},
   overrides = {},
 }) => {
+  console.log(`mounted ${defaultTheme}`);
```
Comment (Cloud11PL) [inline src/theme/ThemeProvider.tsx:40]: should it be here? :D

**Review by Cloud11PL — APPROVED** [review]

File: src/theme/ThemeProvider.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply src/theme/ThemeProvider.tsx:40]: Definitely not

**Review by Droniu — APPROVED** [review]

**Pull request merged**